neuro_san/test/evaluators/agent_evaluator_factory.py

```diff
@@ -21,6 +27,19 @@ class AgentEvaluatorFactory:
     Factory that creates AgentEvaluators
     """
 
+    NAME_TO_AGENT_EVALUATOR: Dict[str, Tuple[Type[AgentEvaluator], bool]] = {
+        "gist": (GistAgentEvaluator, False),
+        "not_gist": (GistAgentEvaluator, True),
+        "greater": (GreaterAgentEvaluator, False),
+        "not_greater": (GreaterAgentEvaluator, True),
+        "keywords": (KeywordsAgentEvaluator, False),
+        "not_keywords": (KeywordsAgentEvaluator, True),
+        "less": (LessAgentEvaluator, False),
+        "not_less": (LessAgentEvaluator, True),
+        "value": (ValueAgentEvaluator, False),
+        "not_value": (ValueAgentEvaluator, True),
+    }
```

Comment:
Switch to a table format.
It's possible at some point we might want to make this user-extendable with other kinds of AgentEvaluators.
We'll see if we need to go there or not.